bot/plugins/chatrank.py

```diff
@@ -165,12 +167,14 @@ def lin_regr(x: Sequence[float], y: Sequence[float]) -> tuple[float, float]:
 
 @command('!chatplot')
 async def cmd_chatplot(config: Config, match: Match[str]) -> str:
-    user = optional_user_arg(match).lower()
+    user_list = optional_user_arg(match).lower().split(' ')
```

Comment:
`.split()` is probably more what you want than `.split(' ')` (double spaces for instance)